tests/by-util/test_mv.rs

```diff
@@ -748,6 +748,19 @@ fn test_mv_errors() {
         .fails()
         .stderr_str()
         .is_empty());
+
+    // $ at.mkdir dir && at.touch file
+    // $ mv -i dir file
+    // err == mv: cannot overwrite non-directory 'file' with directory 'dir'
+    assert!(!scene
```

Comment:
can you please create a separate new test for this?
easier to debug / identify regression

Reply:
I will do it instead :)